x-pack/plugins/apm/server/lib/helpers/get_bucket_size/index.ts

```diff
@@ -10,19 +10,22 @@ import { calculateAuto } from './calculate_auto';
 // @ts-expect-error
 import { unitToSeconds } from './unit_to_seconds';
 
-export function getBucketSize(start: number, end: number, interval: string) {
+export function getBucketSize(
+  start: number,
+  end: number,
+  numBuckets: number = 100
```

Comment:
This is needed for the spark plots, where we should use less buckets because of how small the plot is.